.eslintrc

```diff
@@ -15,12 +15,22 @@
     "ignorePatterns": ["frontend/dist/", "var/"],
     "plugins": ["promise"],
     "settings": {
+        "import/resolver": {
+            "node": {
+              "extensions": [
+                ".js",
+                ".vue",
+                "json"
+              ]
+            }
+        },
+        "import/core-modules": [ "lottie-web-vue" ],
         "node": {
             "allowModules": [
                 "flowforge-test-utils", // exists at test/node_modules/flowforge-test-utils
                 "./config",
                 "promptly",
-                "@core"
+                "lottie-web-vue"
```

Comment:
@Pezmc what is the purpose of the addition of `lottie-web-vue` in this PR?

Reply:
`lottie-web-vue` doesn't export it's module correctly, so the require check fails to find it and throws errors without marking it to ignore.